cmd/import.go

```diff
@@ -54,6 +55,10 @@ var (
 	ErrCollectionShouldExist = fmt.Errorf("collection should exist to import CSV with no field names")
 	ErrNoAppend              = fmt.Errorf(
 		"collection exists. use --append if you need to add documents to existing collection")
+
+	ErrNoRecordsExpected = fmt.Errorf("no records expected in the collection after fixing numbers")
```

Comment:
Is this an error we will show to the user?

Reply:
This should not happen.